operator/gitops/compute/pipeline-service-manager/role.yaml

```diff
@@ -5,12 +5,106 @@ metadata:
   name: pipeline-service-admin
 rules:
   - apiGroups:
+      - ""
+    resources:
+      - namespaces
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - patch
+      - delete
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+      - secrets
+      - serviceaccounts
+      - services
+    verbs:
       - "*"
+  - apiGroups:
+      - apps
     resources:
       - "*"
     verbs:
       - "*"
-  - nonResourceURLs:
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - networking.k8s.io
+    resources:
+      - ingress
+      - networkpolicies
+    verbs:
       - "*"
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+      - deletecollection
+  - apiGroups:
+      - rbac.authorization.k8s.io
+    resources:
+      - roles
+      - rolebindings
+      - clusterroles
+      - clusterrolebindings
+    verbs:
+      - get
+      - list
+      - create
+      - bind
+  - apiGroups:
+      - argoproj.io
+    resources:
+      - applications
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - patch
+      - delete
+      - deletecollection
+  - apiGroups:
+      - tekton.dev
+    resources:
+      - pipelineruns
+      - pipelines
+      - tasks
     verbs:
       - "*"
+  - apiGroups:
+      - pipelinesascode.tekton.dev
+    resources:
+      - repositories
+    verbs:
+      - "*"
```

Comment:
Is there a reason why this is using a wildcard?

Reply:
Yes, commented here - https://github.com/openshift-pipelines/pipeline-service/pull/369#issuecomment-1314923089